docs/opentelemetry-tracing.rst

```diff
@@ -0,0 +1,36 @@
+Tracing with OpenTelemetry
+==================================
+Python-spanner uses `OpenTelemetry <https://opentelemetry.io/>`_ to automatically generates traces providing insight on calls to Cloud Spanner. 
```

Comment:
```suggestion
This library uses `OpenTelemetry <https://opentelemetry.io/>`_ to automatically generates traces providing insight on calls to Cloud Spanner. 
```

The documentation is already associated with the library so it shouldn't need to be stated.